agenta-web/src/hooks/useBlockNavigation.ts

```diff
@@ -13,8 +13,23 @@ const useBlockNavigation = (
     const props = useRef(_props)
     const shouldAlert = useRef(_shouldAlert)
 
+    const beforeUnloadHandler = (event: BeforeUnloadEvent) => {
+        if (blocking.current) {
+            const message = "You have unsaved changes. Are you sure you want to leave?"
```

Comment:
from where we are getting the current modal message then ?
Also shall we remove the other declaration of the modal

Reply:
Browsers have their default message and prioritize it over the set value, this was set as a fallback for older browsers that do not have a default message.